cmake/HalideGeneratorHelpers.cmake

```
@@ -93,8 +93,15 @@ function(add_halide_generator TARGET)
             # TODO: what do we need to do for PACKAGE_NAME PACKAGE_NAMESPACE EXPORT_FILE in this case?
         else ()
             add_executable(${TARGET} ${ARG_SOURCES})
+            target_link_libraries("${TARGET}" PRIVATE Halide::Generator ${ARG_LINK_LIBRARIES})
+            target_include_directories("${TARGET}" PRIVATE "$<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}>")
+
+            # Make a library of the Generator that can be used for (e.g.) cpp_stub.
+            add_library(${TARGET}.objs INTERFACE)
+            target_sources(${TARGET}.objs INTERFACE "$<LIST:FILTER,$<TARGET_OBJECTS:${TARGET}>,EXCLUDE,/GenGen.cpp.o(bj)?$>")
```

The filter here is no longer necessary with the `WHOLE_ARCHIVE` linking that the updated `Halide::Generator` target does.
```suggestion
            target_sources(${TARGET}.objs INTERFACE "$<TARGET_OBJECTS:${TARGET}>")
```